executorlib/cache/shared.py

```
@@ -48,14 +48,17 @@ def done(self) -> bool:
 
 
 def execute_in_subprocess(
-    command: list, task_dependent_lst: list = []
+    command: list,
+    task_dependent_lst: list = [],
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)